### ISSUE TITLE
Pre-trained WGAN generator and discriminator

### ISSUE THREAD

**Issue by matteoghirardelli** [sky4689524/DefenseGAN-Pytorch#1]
Hi, I'm trying to run the code of this repository to implement the same experiments of the original DEFENSE-GAN paper, but with the CIFAR-10 dataset. I was wondering if you have managed to verify the robustness of DEFENSE-GAN on the CIFAR-10 dataset, and if you have any pre-trained models that I could use in order to compare your results with mine. 

Also, is this version of the repository the most recent?

**Comment by sky4689524** [comment]
Hi,
yes I verified the robustness of DEFENSE-GAN on the CIFAR-10 dataset. Unfortunately, I don't have any pre-train models anymore.

yes. It is the recent version.

**Comment by matteoghirardelli** [comment]
How much accuracy did you obtain when reconstructing the original images using the functions in util_defense_GAN.py? I obtained a very low accuracy (around 0.11) for large values of L and R, even by adapting your code to the simpler assessment with the F-MNIST dataset. How many epochs did you use to train the GAN on the CIFAR-10 dataset?Maybe my GAN wasn't trained enough, but it seems to generate reasonable deepfakes, at least with the F-MNIST dataset

**Comment by sky4689524** [comment]
To be honest, I don't remember how much epochs did I train for CIFAR-10 dataset. But I guess you do not need to train a lot of epochs because CIFAR-10 dataset is small size.
